src/v0/destinations/braze/util.js

```diff
@@ -338,24 +373,37 @@ const processBatch = (transformedEvents) => {
       if (Array.isArray(purchases)) {
         purchaseArray.push(...purchases);
       }
+
+      if (Array.isArray(subscription_groups)) {
+        subscriptionsArray.push(...subscription_groups);
+      }
+
+      if (Array.isArray(merge_updates)) {
+        mergeUsersArray.push(...merge_updates);
+      }
+
       successMetadata.push(...transformedEvent.metadata);
     }
   }
   const attributeArrayChunks = _.chunk(attributesArray, 75);
```

Comment:
75, 50 these should be constants defined in config.js

Reply:
updated